### PR TITLE
feat: implemented new KYC step

### PR DESCRIPTION
**Pull request by milan-bc**
## Description (optional)
New KYC required step

## Testing Steps (optional)
Detail the steps required for the reviewer(s) to verify and test these changes.

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/KycRequired/index.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  simpleBuyActions: bindActionCreators(actions.components.simpleBuy, dispatch),
+  identityVerificationActions: bindActionCreators(
+    actions.components.identityVerification,
+    dispatch
+  )
+})
+
+const enhance = compose(connect(mapStateToProps, mapDispatchToProps))
+
+export default enhance(KycRequired)
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/KycRequired/index.tsx:40]: I think maybe you're using an older snippet? Can you update to the latest here? https://github.com/blockchain/blockchain-wallet-v4-frontend/wiki/Helpful-.tsx-Snippets#typescript-react-pure-component-for-remote-types

File: packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/KycRequired/template.tsx [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+`
+
+const Title = styled(Text)`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 7px;
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/KycRequired/template.tsx:29]: Try sticking w/ multiples of 4

File: packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/KycRequired/template.tsx [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+`
+
+const ShortTitleContainer = styled.div`
+  max-width: 154px;
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/KycRequired/template.tsx:33]: Is this needed?

Comment (milan-bc) [reply packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/KycRequired/template.tsx:33]: According to design, they wanted to have it like that in 2 lines, we already have same label in use and I added this to have it into 2 lines for this step

File: packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/KycRequired/template.tsx [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+          fullwidth
+          size='16px'
+          height='48px'
+          nature='primary'
+          data-e2e='handleVerified'
+          onClick={() =>
+            props.identityVerificationActions.verifyIdentity(
+              2,
+              false,
+              'SBEnterAmountCheckout'
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/KycRequired/template.tsx:144]: Can you change this to `KycRequiredStep`

Comment (milan-bc) [reply packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/KycRequired/template.tsx]: @plondon yeah 👍

**Pull request merged**